tests/ignite/handlers/test_checkpoint.py

```diff
@@ -200,8 +204,21 @@ def test_checkpoint_with_dp():
     save_handler.assert_called_with(model.state_dict(), "model_0.pt", metadata)
 
 
-def test_checkpoint_with_global_step_transform():
-    def _test(filename_prefix, to_save, obj, name):
+for prefix in ["", "dummytask"]:
+
+    @pytest.mark.parametrize(
+        "filename_prefix,to_save,obj,name",
+        [
+            (prefix, {"model": model}, model.state_dict(), "model"),
+            (
+                prefix,
+                {"model": model, "optimizer": optimizer},
+                {"model": model.state_dict(), "optimizer": optimizer.state_dict()},
+                "checkpoint",
+            ),
+        ],
+    )
```

Comment:
@divo12 I think you can do 2 `parametrize` decorators instead of for-loop

Reply:
Ok I'll do the changes and commit it